### PR TITLE
USHIFT-1268: add linter for robotframework files

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: test/install.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+ROOTDIR=$(git rev-parse --show-toplevel)
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+RF_VENV="${ROOTDIR}/_output/robotenv"
+
+if [ ! -d "${RF_VENV}" ]; then
+    python3 -m venv "${RF_VENV}"
+    "${RF_VENV}/bin/python3" -m pip install -r "${SCRIPTDIR}/requirements.txt"
+fi
```
Comment (ggiguash) [inline test/install.sh:12]: Can we add this functionality to `scripts/fetch_tools.sh` script to make it consistent with the rest of checks?

**Comment by ggiguash** [comment]
@dhellmann, looks great except for one consistency nit.

File: test/install.sh (hunk 1 above)
Comment (dhellmann) [reply test/install.sh]: Great point, done.

**Comment by dhellmann** [comment]
I think this PR needs https://github.com/openshift/release/pull/37815 to bring in a newer Python 3.

File: scripts/fetch_tools.sh [hunk 2]
```diff
@@ -164,6 +166,27 @@ get_controller-gen() {
     GOBIN=${DEST_DIR} GOFLAGS="" go install sigs.k8s.io/controller-tools/cmd/controller-gen@${ver}
 }
 
+get_robotframework() {
+    local rootdir
+    local venv
+
+    rootdir=$(git rev-parse --show-toplevel)
```
Comment (pacevedom) [inline scripts/fetch_tools.sh:173]: The e2e runner does not move into the microshift root dir, so it does not have git. Can we use the ROOTDIR instead in line 176 and 186?

**Comment by dhellmann** [comment]
Rebased again after #1840 merged.

**Pull request merged**